DOCS/canonical-identity-platform-tutorial-deploy-identity-bundle.md

```diff
@@ -129,35 +106,38 @@ juju config kratos-external-idp-integrator \
   microsoft_tenant_id=<tenant_id>
 ```
 
-After a while the *kratos-external-idp-integrator* status will change to active, this means that Azure AD has been added as a Kratos sign in provider.
+> See more: [Charmhub | Kratos External Idp Integrator](https://charmhub.io/kratos-external-idp-integrator)
+
+After a while the kratos-external-idp-integrator status will change to active, this means that Azure AD has been added as a Kratos sign in provider. Congratulations, your deployment now uses Azure AD as your external identity provider.
 
-# Add SSO to Charmed Grafana 
+# Use Identity Platform to provide SSO
 
-To deploy [Charmed Grafana](https://charmhub.io/grafana-k8s), run:
+Now that our Identity Broker is ready we can use it to provide SSO to an application, for this purpose we are going to use Grafana. To deploy [Charmed Grafana](https://charmhub.io/grafana-k8s), run:
```

Comment:
Here and everywhere else: I'd drop the comma splice structure (https://stlcc.edu/student-support/academic-success-and-tutoring/writing-center/writing-resources/commasplices-runons.aspx#:~:text=Comma%20splices%20and%20run%2Dons%20are%20major%20sentence%20errors%20that,at%20all%20separates%20independent%20sentences.). So, I'd rewrite:

> Now that our Identity Broker is ready, we can use it to provide SSO to an application. For this purpose we are going to use Grafana.

Reply:
TIL! thanks